crates/swc_ecma_transforms_compat/src/es2017/async_to_generator.rs

```diff
@@ -172,17 +169,15 @@ impl Fold for Actual {
                 type_params: Default::default(),
                 return_type: Default::default(),
             },
-            ..m
+            key: m.key.take(),
```

Comment:
Is this required?
If `rustc` complains about it, it would be better to assign to `m.function` instead as this is useless `memmove`.